templates/blog/post_create.html

```diff
@@ -0,0 +1,23 @@
+% extends 'base.html' %}

```

Comment:
There is a missing opening curly brace in the `{% extends 'base.html' %}` tag. It should be `{% extends 'base.html' %}`.